src/sidebar/util/state.js

```diff
@@ -1,11 +1,14 @@
+/** @typedef {import('redux').Store} Store */
```

Comment:
This is an example of using types that are already included in a third-party package.